src/components/Player/Pages/Counts/Counts.jsx

```diff
@@ -13,14 +14,18 @@ import styles from './Counts.css';
 const Counts = ({ playerId, counts, getPlayerCounts }) => (
   <div>
     <TableFilterForm submitAction={getPlayerCounts} id={playerId} page="counts" />
-    <div className={styles.countsContainer}>
-      {Object.keys(counts).map((key, index) => (
-        <div key={index} className={styles.countTable}>
-          <TableContainer title={deSnake(key)} />
-          <Table paginated columns={playerCountsColumns} data={counts[key].list} />
-        </div>
-      ))}
-    </div>
+    <Row>
+      <div className={styles.countsContainer}>
+        {Object.keys(counts).map((key, index) => (
+          <Col xs={12} sm={12} md={6} lg={4} key={index}>
+            <div className={styles.countTable}>
+              <TableContainer title={deSnake(key)} />
+              <Table paginated columns={playerCountsColumns} data={counts[key].list} />
```

Comment:
screenshots look great. Here's what I would ask:
1. This table should be wrapped in TableContainer. I don't know why I didn't do it like that in the first place, but it should be like this:
```js
<TableContainer title={deSnake(key)}>
  <Table paginated columns={playerCountsColumns} data={counts[key].list} />
</TableContainer>
```

2. put the row and column into TableContainer
3. Pass xs sm md and lg props into TableContainer. Make default values 12 (so we don't have to pass them in to get max width).